acceptance/tests/vault/vault_wan_fed_test.go

```diff
@@ -517,6 +521,19 @@ func TestVault_WANFederationViaGateways(t *testing.T) {
 
 	logger.Log(t, "checking that connection is successful")
 	k8s.CheckStaticServerConnectionSuccessful(t, primaryCtx.KubectlOptions(t), StaticClientName, "http://localhost:1234")
+
+	if cfg.NoCleanupWanFed {
+		//time.Sleep(1800 * time.Second)
+		intVar, err := strconv.Atoi(cfg.TestDuration)
```

Comment:
I don't think you want to use TestDuration as a timeout. Instead:
 
 a) Remove the TestDuration flag.
 b) Add a retry that checks for something successful or several steps that are successful.
 
 If these tests take over 2 hours to run then we will not be able to run them in Github Actions as the runners will timeout...

Reply:
@curtbushko - I left a PR comment which is sorta in reply to your comments here. If you don't mind maybe have a look at it and see if that makes any sense?